config.go

```diff
@@ -132,3 +133,19 @@ func (c *Config) Slog() *slog.Logger {
 	}
 	return slog.New(slog.NewTextHandler(os.Stderr, opts))
 }
+
+func (c *Config) fieldMapToPlaceKey() func(groups []string, a slog.Attr) slog.Attr {
+	fieldMap, ok := c.Formatter.Data["fieldmap"].(map[string]interface{})
```

Comment:
Are c.Formatter and c.Formatter.Data never nil?

Reply:
Also, is fieldmap always lowercase?

Reply:
c.Formatter can't be nil, since we already access it before.
Data can be nil, but won't matter as an nil map is equivalent to an empty map (at least to read)
and yes viper / mapstructure will always return lowercase